htdocs/core/modules/bank/doc/pdf_ban.modules.php

```diff
@@ -388,6 +388,6 @@ protected function _pagefoot(&$pdf, $object, $outputlangs, $hidefreetext = 0)
 		global $conf;
 
 		$showdetails = empty($conf->global->MAIN_GENERATE_DOCUMENTS_SHOW_FOOT_DETAILS) ? 0 : $conf->global->MAIN_GENERATE_DOCUMENTS_SHOW_FOOT_DETAILS;
-		//return pdf_pagefoot($pdf,$outputlangs,'BANK_FREE_TEXT',$this->emetteur,$this->marge_basse,$this->marge_gauche,$this->page_hauteur,$object,$showdetails,$hidefreetext);
+		return pdf_pagefoot($pdf,$outputlangs,'BANK_FREE_TEXT',$this->emetteur,$this->marge_basse,$this->marge_gauche,$this->page_hauteur,$object,$showdetails,$hidefreetext);
```

Comment:
This fix the warning but it also change the behaviour by calling a method that was not called.
Call is to remove warning only.
So i suggest instead to keep the comment and add a return 1